<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="8" passed="7">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="22091" started-at="2019-02-05T08:56:12Z" finished-at="2019-02-05T08:56:34Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="22091" started-at="2019-02-05T08:56:12Z" finished-at="2019-02-05T08:56:34Z">
      <class name="com.training.functional.tests.FilterOrderTest_RTTC047">
        <test-method status="PASS" signature="setUpBeforeClass()[pri:0, instance:com.training.functional.tests.FilterOrderTest_RTTC047@3e6fa38a]" name="setUpBeforeClass" is-config="true" duration-ms="16" started-at="2019-02-05T08:56:12Z" finished-at="2019-02-05T08:56:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUpBeforeClass -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.training.functional.tests.FilterOrderTest_RTTC047@3e6fa38a]" name="setUp" is-config="true" duration-ms="7004" started-at="2019-02-05T08:56:12Z" finished-at="2019-02-05T08:56:19Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="LoginTest()[pri:0, instance:com.training.functional.tests.FilterOrderTest_RTTC047@3e6fa38a]" name="LoginTest" duration-ms="2907" started-at="2019-02-05T08:56:19Z" finished-at="2019-02-05T08:56:22Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method status="PASS" signature="movetoSales()[pri:1, instance:com.training.functional.tests.FilterOrderTest_RTTC047@3e6fa38a]" name="movetoSales" duration-ms="1505" started-at="2019-02-05T08:56:22Z" finished-at="2019-02-05T08:56:23Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- movetoSales -->
        <test-method status="PASS" signature="validOrderTestWithId()[pri:2, instance:com.training.functional.tests.FilterOrderTest_RTTC047@3e6fa38a]" name="validOrderTestWithId" duration-ms="2067" started-at="2019-02-05T08:56:23Z" finished-at="2019-02-05T08:56:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validOrderTestWithId -->
        <test-method status="PASS" signature="validOrdertestwithname()[pri:3, instance:com.training.functional.tests.FilterOrderTest_RTTC047@3e6fa38a]" name="validOrdertestwithname" duration-ms="1886" started-at="2019-02-05T08:56:25Z" finished-at="2019-02-05T08:56:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validOrdertestwithname -->
        <test-method status="PASS" signature="validOrdertestbyStatus()[pri:4, instance:com.training.functional.tests.FilterOrderTest_RTTC047@3e6fa38a]" name="validOrdertestbyStatus" duration-ms="1645" started-at="2019-02-05T08:56:27Z" finished-at="2019-02-05T08:56:29Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validOrdertestbyStatus -->
        <test-method status="PASS" signature="validOrdertestbyDtAd()[pri:5, instance:com.training.functional.tests.FilterOrderTest_RTTC047@3e6fa38a]" name="validOrdertestbyDtAd" duration-ms="1383" started-at="2019-02-05T08:56:29Z" finished-at="2019-02-05T08:56:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validOrdertestbyDtAd -->
        <test-method status="PASS" signature="validOrdertestbyDtMd()[pri:6, instance:com.training.functional.tests.FilterOrderTest_RTTC047@3e6fa38a]" name="validOrdertestbyDtMd" duration-ms="1762" started-at="2019-02-05T08:56:30Z" finished-at="2019-02-05T08:56:32Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validOrdertestbyDtMd -->
        <test-method status="FAIL" signature="validOrderTestbyTotal()[pri:7, instance:com.training.functional.tests.FilterOrderTest_RTTC047@3e6fa38a]" name="validOrderTestbyTotal" duration-ms="5" started-at="2019-02-05T08:56:32Z" finished-at="2019-02-05T08:56:32Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.training.functional.tests.FilterOrderTest_RTTC047.validOrderTestbyTotal(FilterOrderTest_RTTC047.java:126)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:580)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:716)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:988)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validOrderTestbyTotal -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.training.functional.tests.FilterOrderTest_RTTC047@3e6fa38a]" name="tearDown" is-config="true" duration-ms="1855" started-at="2019-02-05T08:56:32Z" finished-at="2019-02-05T08:56:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.training.functional.tests.FilterOrderTest_RTTC047 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
